.github/workflows/integration-tests.yml

```diff
@@ -18,7 +18,7 @@ jobs:
           aws-access-key-id: ${{ secrets.BOT_AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.BOT_AWS_SECRET_ACCESS_KEY }}
           aws-region: us-east-1
-      - run: npm install @boostercloud/cli@0.15.1 --global
+      - run: npm install @boostercloud/cli@0.16.1 --global
```

Comment:
Not about this PR, but can I get a pointer to why we need to install the Booster CLI globally for the integration tests?

Reply:
@NickSeagull this is the integration for the rocket, so it's required to use boost deploy from the cli.